present/slideshow.py

```diff
@@ -189,6 +214,14 @@ def play(
 
                 a = time.time()
                 self.screen.draw_next_frame(repeat=repeat)
+
+                if self.current_slide < len(self.slides) and (
+                    code_blocks := self.slides[self.current_slide].code_blocks
+                ):
+                    for tpl in code_blocks:
+                        block_code, row = tpl
+                        render_code_block(self.screen, block_code, row)
```

Comment:
It doesn't seem to matter if `render_code_block()` happens before or after `self.screen.draw_next_frame()`. This just seemed like a good place to render the highlighted block.